src/task5/task1/Figure.java

```diff
@@ -0,0 +1,29 @@
+package task5.task1;
+
+public abstract class Figure {
+    int x;
+    int y;
+
+    abstract double square();
```

Comment:
As i understand correctly you should to realize abstarct method "square" which should calculates the area of a shape.